driver-core/src/test/functional/com/mongodb/client/model/AggregatesFunctionalSpecification.groovy

```diff
@@ -189,4 +191,32 @@ class AggregatesFunctionalSpecification extends OperationFunctionalSpecification
         containsAny([a, b, c], aggregate([sample(1)]).first())
     }
 
+
+    @IgnoreIf({ !serverVersionAtLeast(asList(3, 1, 8)) || !isEnterpriseServer() })
+    def '$lookup'() {
+        given:
+        def fromCollectionName = 'lookupCollection'
+        def fromHelper = getCollectionHelper(new MongoNamespace(getDatabaseName(), fromCollectionName))
+
+        getCollectionHelper().drop()
```

Comment:
This gets dropped in base class setup, so no need to do it here as well I think.